pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -169,6 +171,68 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "v1beta2 with non-default global percentageOfNodesToScore",
+			data: []byte(`
+apiVersion: kubescheduler.config.k8s.io/v1beta2
+kind: KubeSchedulerConfiguration
+percentageOfNodesToScore: 10
+`),
+			wantProfiles: []config.KubeSchedulerProfile{
+				{
+					SchedulerName:            "default-scheduler",
+					PercentageOfNodesToScore: nil,
+					Plugins:                  defaults.PluginsV1beta2,
+					PluginConfig: []config.PluginConfig{
```

Comment:
```suggestion
					PluginConfig: defaults.PluginConfigsV1beta2,
```

And similarly for the rest of the tests

Reply:
It looks like this was reverted.